database_admin/migrations/41_partition_packages.up.sql

```diff
@@ -0,0 +1,62 @@
+ALTER TABLE system_package
```

Comment:
File should start with 3-digit i.e. 041-

Reply:
Fixed